src/redux/features/categories/categoriesSlice.js

```diff
@@ -0,0 +1,14 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = { categories: [] };
+
+const categoriesSlice = createSlice({
+  name: 'category',
+  initialState,
+  reducers: {
+    status: () => 'Under Construction',
+  },
```

Comment:
- [ ] Ouch! this is a close one but as specified in the [project requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/add_reducers_an_actions_v2.md) please make sure to first check the status (from the action payload) before returning the string, take a look at the code snippet shown below:
```jsx
      state.categories =
        action.payload === "Under construction"
          ? "Under construction"
          : state.categories
```

Reply:
@AbelG101
 Hello Abel,
I'm Blessing. Thank you for the review. Please kindly throw more light as I'm confused on where to add the suggested code you requested. I look forward to hearing from you. Thank you.